### PR TITLE
Disable download token checking by default

### PR DESCRIPTION
**Pull request by moio**
## What does this PR change?

It disables token checking by default on Servers, which is convenient to consume repos from other sumaform hosts.

### CODE REVIEW

File: README_ADVANCED.md [hunk 1]
```diff
@@ -114,6 +114,7 @@ By default, sumaform deploys hosts with a range of tweaked settings for convenie
    * `create_sample_activation_key`: whether to create a sample activation key. Requires `create_first_user`
    * `create_sample_bootstrap_script`: whether to create a sample bootstrap script for traditional clients. Requires `create_sample_activation_key`
    * `publish_private_ssl_key`: copies the private SSL key in /pub for Proxies to copy automatically. Set to `false` for manual distribution
+   * `disable_download_tokens`: disable package token download checks. Set to `true` to enable checking
```
Comment (srbarrios) [inline README_ADVANCED.md]:
```suggestion
   * `disable_download_tokens`: disable package token download checks. Set to `true` to disable checking
```

**Comment by srbarrios** [comment]
Actually, if we will change all our main.tf to set as false this new parameter. Even in the examples.
It could be better to set as default to false?

File: README_ADVANCED.md (hunk 1 above)
Comment (moio) [reply README_ADVANCED.md]: :man_facepalming:

...embarassing!

**Comment by moio** [comment]
Well, the defaults apply to developers/debuggers/users that want to demo SUSE Manager/Uyuni and are optimized towards convenience. In this case, default is to disable token checking as that's more convenient than enabling it (which is the SUSE Manager default).

In testsuites this is different: we want to test the software in the way it's configured by default, unchanged. That's why the testsuite examples have `false`.

**Comment by srbarrios** [comment]
> Well, the defaults apply to developers/debuggers/users that want to demo SUSE Manager/Uyuni and are optimized towards convenience. In this case, default is to disable token checking as that's more convenient than enabling it (which is the SUSE Manager default).
> 
> In testsuites this is different: we want to test the software in the way it's configured by default, unchanged. That's why the testsuite examples have `false`.

Fair enough then :)

**Pull request merged**